proposed/2022/dotnet-nuget-config-spec.md

```diff
@@ -0,0 +1,369 @@
+# NuGet configuration CLI for dotnet.exe
+
+- Author: [Heng Liu](https://github.com/heng-liu)
+- GitHub Issue [8420](https://github.com/NuGet/Home/issues/8420)
+
+## Problem Background
+
+Currently, there is no NuGet configuration CLI for dotnet.exe. It's inconvenient for NuGet users to know about the NuGet configuration file locations and figure out where is the merged configuration coming from. 
+
+## Who are the customers
+
+This feature is for dotnet.exe users.
+
+## Goals
+Design and implement `dotnet nuget config` command.
+
+## Non-Goals
+Design and implement in the same way for NuGet.exe command.
+
+## Solution
+The following command will be implemented in the `dotnet.exe` CLI.
+
+### `dotnet nuget config`
+
+### Commands
+
+### Paths
+
+List all the paths of NuGet configuration files that will be applied, when invoking NuGet command from the current working directory path.
+
+The listed NuGet configuration files are in priority order. So the order of loading those configurations is reversed, that is, loading order is from the bottom to the top. So the configuration on the top will apply.
+You may refer to [How settings are applied](https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied) for more details. 
+
+#### Arguments
+
+- WORKING_DIRECTORY
+
+Run this command as if working directory is set to the specified directory. If it's not specified, the current working directory will be used.
```

Comment:
If can choose to not pass anything isn't this one is `option` rather than `argument`?

Reply:
Not exactly. Here is the definition of [arguments](https://learn.microsoft.com/en-us/dotnet/core/tools/#arguments) and [options](https://learn.microsoft.com/en-us/dotnet/core/tools/#options) in dotnet command world.  
An example of argument is a solution or a project. it can be optional as well.
The folder path is something similar with a solution or a project, so we choose to use it as an argument. We checked this with  .NET sdk team in the review meetings.